app/src/main/res/layout/dialog_set_status.xml

```diff
@@ -462,4 +462,11 @@
 
     </LinearLayout>
 
+    <androidx.emoji2.emojipicker.EmojiPickerView
+        android:id="@+id/emoji_picker"
+        android:layout_width="match_parent"
+        android:visibility="gone"
+        android:layout_height="match_parent"
+        app:emojiGridColumns="9" />
```

Comment:
app:emojiGridColumns="9" is too much here. The emojis are too small on my device (Galaxy a32).
If it can't be flexible depending of the available width, 7 columns worked for me best on this device.
But this might need some tinkering to look good on all devices.

Reply:
https://developer.android.com/guide/topics/large-screens/support-different-screen-sizes

Would this be a better solution @mahibi ???
<img width="932" alt="Screenshot 2023-11-18 at 6 13 56 PM" src="https://github.com/nextcloud/talk-android/assets/99678760/58ed1c5f-e9dc-4a7d-8845-498576bf8fa0">
<img width="586" alt="Screenshot 2023-11-18 at 6 14 29 PM" src="https://github.com/nextcloud/talk-android/assets/99678760/3884dd8f-f941-4c9f-8e66-b65a59ca2641">

Reply:
sorry, oversaw this..
sure, calculating on screen values would work.
But we can also go with a fixed value for now and gain some feedback on test releases (this PR won't be available for 18.0.0, so enough time get feedback..).
If people complain, the calculation can be added later..

Reply:
sure, lets set 7 for now